src/components/notes/cards/All.tsx

```diff
@@ -0,0 +1,34 @@
+/** Vendor */
+import React from "react";
+
+/** Redux */
+import { IRootState } from "@redux/configureStore";
+import { useSelector } from "react-redux";
+
+/** Custom */
+import Note from "./Note";
+import { INote } from "../../../types/note";
```

Comment:
Ideally you create a file in `types/index.ts` and write `export type { INote } from "./note"` in it. This way everywhere in your application you can import all types from `types` instead of having to explicitly say `types/note` or specify the file directly from where you want to import the type from.

Reply:
I think I see what you are saying.